### PR TITLE
Made the kanvas logo responsive

### LINKED ISSUE #6069: Responsive issue on homepage
### Description

on homepage https://layer5.io
Screenshot_20241108-173614

### Environment:
- Host OS:
- Browser:

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by NishantSinghhhhh**
Description

This PR fixes #6069

Notes for Reviewers:
This PR makes the logo of bigger size 
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Kanvas/kanvas_banner.js [hunk 1]
```diff
@@ -96,11 +96,13 @@ const BannerSectionWrapper = styled.div`
         overflow: hidden;
         margin-top: -7rem;
         position: relative;
-        left: 30%;
+        left: 11%;
+        margin-top: -7rem;
```
Comment (vishalvivekm) [inline src/sections/Kanvas/kanvas_banner.js:100]:
```suggestion
```

Comment (vishalvivekm) [reply src/sections/Kanvas/kanvas_banner.js:100]: the style is already look 3 lines above

File: src/sections/Kanvas/kanvas_banner.js [hunk 2]
```diff
@@ -117,22 +119,20 @@ const BannerSectionWrapper = styled.div`
 
     @media screen and (max-width: 1700px) {
         .kanvas-logo {
-            width: 12rem;
-            /* right: 12rem; */
-            /* margin-top: -5rem; */
+        width: 20rem;
+        left: 5rem;
+        margin-top: -7rem;
```
Comment (vishalvivekm) [inline src/sections/Kanvas/kanvas_banner.js:124]:
```suggestion
```

Comment (vishalvivekm) [reply src/sections/Kanvas/kanvas_banner.js:124]:
```css
margin-top: -7rem;
```

is by default for the `.kanvas-logo`, unless we're modifying the value, writing it again is redundant.

**Comment by l5io** [comment]
🚀 Preview for commit 7a2003268176ccef6e73a21172517cedd5ea5a9b at: https://6730e2c2a9cb37e30ceca63c--layer5.netlify.app

**Comment by NishantSinghhhhh** [comment]
@vishalvivekm , I have done the requested changes

**Comment by l5io** [comment]
🚀 Preview for commit 3e998c372785d096a8e3a8598414d6d0daab43f1 at: https://6730e708e3f337e7b9e4a264--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@NishantSinghhhhh please take a look at : https://github.com/layer5io/layer5/pull/6070

**Comment by vishalvivekm** [comment]
@NishantSinghhhhh Thank you for your contribution!
Let's discuss this during the website call on Monday at 5:30 PM IST (7:00 AM CT).

Please add it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit?usp=drivesdk).

**Comment by sudhanshutech** [comment]
@NishantSinghhhhh any screenshot?

**Review by leecalcote — CHANGES_REQUESTED** [review]
This doesn't appear to fix either issue on the home page or the kanvas page. Am I missing something, @NishantSinghhhhh?
Screenshot 2024-12-09 at 2 36 02 PM

**Comment by leecalcote** [comment]
PR abandoned. Fix provided elsewhere.

**Pull request closed without merging**